### ISSUE TITLE
THArgCheck doesn't print error message from interpreter

### ISSUE THREAD

**Issue by nicholas-leonard** [torch/trepl#11]
To reproduce bug:
```bash
$> th
a = torch.randn(3,4,1)
b = torch.randn(2,4,1)
i = torch.randperm(1):long()
a:indexCopy(1, i, b)
```
Run the same thing in a lua scritp to get a nice error message.

Makes it impossible to debug using the th shell.

**Comment by soumith** [comment]
fixed via #12

**Comment by nicholas-leonard** [comment]
thanks guys. Works great.
